app/services/paper_converter.rb

```diff
@@ -21,4 +19,14 @@ def self.connection
       config.adapter :net_http
     end
   end
+
+  def self.post_ihat_job(file)
+    connection.post("/jobs", job: {
+      input: file,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.